spec/fortnox/api/repositories/examples/only.rb

```diff
@@ -30,13 +30,11 @@ def repository_only(repository, vcr_cassette, filter)
     end
 
     context 'with invalid filter' do
-      subject do
-        when_performing do
+      it do
+        expect do
```

Comment:
I REALLY don't agree with using a `do ... end` block when you are chaining. That is just bad form. It should at least be:
```
it 'does something here' do
  expect{ repository_only(repository, 'filter_invalid', 'doesntexist') }.to raise_error(Fortnox::API::RemoteServerError, /ogiltigt filter/)
end
```